packages/react/src/categories/hooks/useCategory.ts

```diff
@@ -0,0 +1,49 @@
+import {
+  fetchCategory as fetchCategoryAction,
+  getCategory,
+  getCategoryError,
+  isCategoryFetched,
+  isCategoryLoading,
+  StoreState,
+} from '@farfetch/blackout-redux';
+import { useAction } from '../../helpers';
+import { useCallback, useEffect } from 'react';
+import { useSelector } from 'react-redux';
+import type { UseCategoryOptions } from './types/useCategory';
+
+const useCategory = (id: number, options: UseCategoryOptions = {}) => {
```

Comment:
What do you think of this change?

```suggestion
const useCategory = (categoryId: Category['id'], options: UseCategoryOptions = {}) => {
```
I think `categoryId` is a better name for that parameter and the type can be set to `Category['id']`. What do you think?